src/utils.zig

```diff
@@ -2099,6 +2099,10 @@ pub fn castFunctionReturnToAnyopaque(function: anytype) *const CastFunctionRetur
 	return @ptrCast(&function);
 }
 
+pub fn format(allocator: NeverFailingAllocator, comptime fmt: []const u8, args: anytype) []u8 {
```

Comment:
As mentioned on another PR, if you want this, please make an issue for it first.

Reply:
Boy scout rule. We already have issue for that #920

Reply:
That's specifically about file paths, which should be specialized.

Reply:
Here #1595